e2e/ansible/playbooks/utils/namespace_task.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+- block:
+
+   - name: Create test specific namespace.
+     shell: source ~/.profile;  kubectl create ns {{ ns }}
+     args:
+      executable: /bin/bash
+     delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+   - name: Checking the status  of test specific namespace.
+     shell: source ~/.profile; kubectl get ns | grep {{ ns }} | awk {'print $2'}
+     args:
+      executable: /bin/bash
+     register: npstatus
+     delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+     until: "'Active' in npstatus.stdout"
+     delay: 30
+     retries: 10
+
+  when: status == "create"
+
+
+
+- block:
+
+
+   - name: Deletion of test specific  namespace.
+     shell: source ~/.profile; kubectl delete ns {{ ns }}
+     args:
+       executable: /bin/bash
+     delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+   - name: Check the namespace status.
+     shell: source ~/.profile; kubectl get ns
+     args:
+      executable: /bin/bash
+     register: npdel
+     delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+     until: "'ns' not in npdel.stdout"
```

Comment:
This will equate to a string called "ns" @vibhor995 !

Reply:
@ksatchit  it was giving some warning message related to jinja templates so in order to keep the clean run of playbook I put it as ns, not {{ns}}. @dargasudarshan

Reply:
@vibhor995 , there is no need to provide curly braces for the variables/jinja templates in conditional statements. Just *ns* should do. Adding single quotes will make it a strict string compare.